modules/navigation.js

```diff
@@ -0,0 +1,33 @@
+// Select Nav contents
+export const nav = document.querySelector('#nav-bar');
+export const list = document.getElementById('list');
+export const add = document.getElementById('add');
+export const contact = document.getElementById('contact');
+
+// Select Sections
+const listSection = document.querySelector('#book-list');
+const addBookSection = document.querySelector('.add-books');
+const contactSection = document.querySelector('#contact-us');
+
+// Create Display Functions
+
+// Show Only List of Books page
+export function showList() {
+  listSection.style.display = 'block';
+  addBookSection.style.display = 'none';
+  contactSection.style.display = 'none';
+}
+
+// Show Only Add Books page
+export function showBooks() {
+  listSection.style.display = 'none';
+  addBookSection.style.display = 'block';
+  contactSection.style.display = 'none';
+}
+
+// Show Contact Us page
+export function showContactUs() {
+  listSection.style.display = 'none';
+  addBookSection.style.display = 'none';
+  contactSection.style.display = 'block';
+}
```

Comment:
Good job here... But,

- [ ] It would be nice if you could change these functions to using the arrow function syntax
- [ ] It was required you use the arrow function syntax. Kindly work on to that change 👍 

```mdx
export const showList = () => {
  //
};

export const showBooks = () => {
  //
};

export const showContactUs = () => {
   //
};
```